src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.component.html

```diff
@@ -87,5 +87,11 @@ <h6>
         <h5><span jhiTranslate="artemisApp.fileUploadExercise.assessedSubmission">Your assessed submission</span>:</h5>
         <jhi-file-upload-result [result]="result"></jhi-file-upload-result>
     </div>
-    <jhi-complaint-interactions *ngIf="fileUploadExercise" [result]="result" [participation]="participation" [exercise]="fileUploadExercise"> </jhi-complaint-interactions>
+    <jhi-complaint-interactions
+        *ngIf="fileUploadExercise && fileUploadExercise.course?.complaintsEnabled"
```

Comment:
I am wondering if we should move this check inside the `<jhi-complaint-interactions>` component. From the usage of this component in this PR, it looks like this is 100% needed, so we could simplify the API.